yangcatalog-ui/src/app/shared/notification.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file will be accessible at http://<BASE_URL>/notifications/notification.json
Rename it to `notification-example.json` or remove it completely.